src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyClaimantJudgmentVariedDeterminationOfMeansNotificationHandler.java

```diff
@@ -0,0 +1,106 @@
+    public static final String TASK_ID = "NotifyClaimantJudgmentVariedDeterminationOfMeans";
+    private static final String REFERENCE_TEMPLATE =
+        "claimant-judgment-varied-determination-of-means-%s";
+
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+    private final OrganisationService organisationService;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
+            callbackKey(ABOUT_TO_SUBMIT),
+            this::notifyClaimantJudgmentVariedDeterminationOfMeans
+        );
+    }
+
+    @Override
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private CallbackResponse notifyClaimantJudgmentVariedDeterminationOfMeans(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        if (caseData.getApplicantSolicitor1UserDetails().getEmail() != null) {
+            notificationService.sendMail(
+                caseData.getApplicantSolicitor1UserDetails().getEmail(),
+                getTemplate(),
+                addProperties(caseData),
+                getReferenceTemplate(caseData)
+            );
+        }
+
+        return AboutToStartOrSubmitCallbackResponse.builder().build();
+    }
+
+    @Override
+    public Map<String, String> addProperties(CaseData caseData) {
+        String defendantName = "";
+        if (getMultiPartyScenario(caseData).equals(ONE_V_ONE)
+            || getMultiPartyScenario(caseData).equals(TWO_V_ONE)) {
+            defendantName = getPartyNameBasedOnType(caseData.getRespondent1());
+        } else {
+            defendantName = getPartyNameBasedOnType(caseData.getRespondent1())
+                .concat(" and ")
+                .concat(getPartyNameBasedOnType(caseData.getRespondent2()));
+        }
+        return Map.of(
+                CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+                LEGAL_ORG_NAME, getApplicantLegalOrganizationName(caseData),
+                DEFENDANT_NAME,  defendantName
+            );
+    }
+
+    private String getTemplate() {
+        return notificationsProperties.getNotifyClaimantJudgmentVariedDeterminationOfMeansTemplate();
+    }
+
+    private String getReferenceTemplate(CaseData caseData) {
+        return String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference());
+    }
+
+    public String getApplicantLegalOrganizationName(CaseData caseData) {
+        String id = caseData.getApplicant1OrganisationPolicy().getOrganisation().getOrganisationID();
+        Optional<Organisation> organisation = organisationService.findOrganisationById(id);
+        return organisation.isPresent() ? organisation.get().getName() :
+            caseData.getApplicantSolicitor1ClaimStatementOfTruth().getName();
+    }
```

Comment:
Think we can move the getApplicantLegalOrganizationName() to a common utils class so that i can be reused?